### ISSUE TITLE
Disclaimer on report page

### ISSUE THREAD

**Issue by emacsen** [sbagroy986/openstreetmap-website#2]
Let's make a report page text that talks about why you should or shouldn't report an item

**Comment by tomhughes** [comment]
You're talking about the bit that currently just says "placeholder" right? This bit:

![placeholder](https://cloud.githubusercontent.com/assets/147741/9232244/8cfb4a40-4124-11e5-8d74-c6e29a1b3bfe.png)

**Comment by sbagroy986** [comment]
Yes. We just need to discuss the strings to replace the placeholders and then I'll fix this.
